config/tests/scenario_config_quick.csv

```diff
@@ -0,0 +1,2 @@
+title;start;cm_nash_mode;cm_iteration_max;optimization;results_folder;output;cm_quick_mode;force_replace;slurmConfig
```

Comment:
I think, setting `output` to `NA` does not do what is intended. At least the `quick` run shows
```
Starting output generation for output/testOneRegi


Finished output generation for output/testOneRegi!
```
instead of [this line](https://github.com/remindmodel/remind/blob/06e35960adf1d6fc6a3bab0264a0517d317c12fe/output.R#L214) catching this with `is.na(output)`.

[this line](https://github.com/remindmodel/remind/blob/develop/output.R#L214)

Reply:
It is caught instead by [this line](https://github.com/remindmodel/remind/blob/06e35960adf1d6fc6a3bab0264a0517d317c12fe/output.R#L223) because `NA.R` does not exist. Maybe good enough?

Reply:
Personally, I would prefer the following:
```
if (all(is.na(output) | output == "NA")) {
  message("\nNo output generation, as output was set to NA, as for example for --testOneRegi or --quick.")
```
I was very confused when I started that testOneRegi did not produce the mif file as output and didn't understand why…